Recast/Cells/HomeContinueListeningCollectionViewCell.swift

```diff
@@ -9,25 +9,22 @@
 import UIKit
 import SnapKit
 
-/// UITableViewCell subclass to be used for displaying a "Continue Listening" collection view inside a table view
-class ContinueListeningTableViewCell: UITableViewCell {
+/// UICollectionViewCell subclass to be used for displaying a "Continue Listening" collection view inside a table view
+//swiftlint:disable:next type_name
+class HomeContinueListeningCollectionViewCell: UICollectionViewCell {
     var collectionView: UICollectionView!
     private let continueCvReuse = "continueListeningCvReuse"
 
     // MARK: Lifecycle
-    override init(style: UITableViewCellStyle, reuseIdentifier: String?) {
-        super.init(style: style, reuseIdentifier: reuseIdentifier)
+    override init(frame: CGRect) {
```

Comment:
Does CollectionViewCell not have a reuseIdentifier init? If it does we should use that.